teuthology/templates/rocket-sleep-before-teardown.jinja2

```diff
@@ -0,0 +1,6 @@
+The teuthology job [{{ job_id }}]({{ job_info }}) for suite *{{ suite_name }}* owned by '{{ owner }}' has fallen asleep with status '{{ status }}' at {{ sleep_date }} for __{{ sleep_time }}__ ({{ sleep_time_sec }} seconds).
+Open [teuthology.log]({{ job_logs }}teuthology.log) for details, or go to [all logs]({{ job_logs}}).
+
+Job Description: {{ job_desc }}
+Run Name: {{ run_name }}
+Task Stack: {{ task_stack }}
```

Comment:
I suggest as a less ambiguous name for this file: `rocket-chat-sleep-before-teardown.jinja2` or simply `rocket-chat-notification.jinja2` (since this is the only type of Rocket.Chat notification in teuthology, there is no need to distinguish it from other types)

Reply:
Thanks for the suggestion, I would prefer leave `sleep-before-teardown` suffix because of possible adding another template for run completion rocket chat notification. Didn't add this in this PR because I have no time for it yet.

Reply:
Changed the `rocket-` to `rocketchat-`, Danke sehr.